integration/spark2/src/test/scala/org/apache/carbondata/spark/testsuite/partition/TestAlterPartitionTable.scala

```diff
@@ -466,6 +467,111 @@ class TestAlterPartitionTable extends QueryTest with BeforeAndAfterAll {
     checkAnswer(result_after5, result_origin5)
   }
 
+   test("test exception when alter partition and the values"
```

Comment:
Multiple tests under single test definition.split it into multiple

Reply:
actually I refer to carbondata-spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/partition/TestDDLForPartitionTable.scala. All the new added test cases are belong to the same type.

Reply:
That is also wrong.you can rework both and split into multiple test cases in another pr